tools/razzle.cmd

```diff
@@ -84,6 +20,34 @@ set OPENCON=%OPENCON_TOOLS:~0,-7%
 rem Add nuget to PATH
 set PATH=%PATH%%OPENCON%\dep\nuget;
 
+rem Run nuget restore so you can use vswhere
+nuget restore %OPENCON% -Verbosity quiet
+
+rem Find vswhere
+rem from https://github.com/microsoft/vs-setup-samples/blob/master/tools/vswhere.cmd
+for /f "usebackq delims=" %%I in (`dir /b /aD /o-N /s "%~dp0..\packages\vswhere*" 2^>nul`) do (

```

Comment:
Is there a reason this needs to be a batch file rather than using PowerShell? I imagine this could be clearer than the obscure `dir` command if it used PowerShell.

Reply:
I was just following what was already there.

Reply:
Because not everyone likes Powershell. Our dev team is split between users of Powershell and cmd, so we have scripts for both.